core/commons/che-core-commons-lang/src/main/java/org/eclipse/che/commons/lang/ZipUtils.java

```diff
@@ -1,27 +1,26 @@
 /*
- * Copyright (c) 2012-2018 Red Hat, Inc.
- * This program and the accompanying materials are made
- * available under the terms of the Eclipse Public License 2.0
- * which is available at https://www.eclipse.org/legal/epl-2.0/
+ * Copyright (c) 2012-2018 Red Hat, Inc. This program and the accompanying materials are made
```

Comment:
I believe this changes should in the license are made by mistake.

Reply:
Hmh...I might have had "skip license check" flag on when building this branch. Good catch, I'll fix it.